service/importer/assets/schema.json

```diff
@@ -90,14 +90,26 @@
                   "type": "boolean"
                 },
                 "type": {
-                  "type": "string",
-                  "enum": [
-                    "String",
-                    "Number",
-                    "Boolean",
-                    "Object"
-                  ],
-                  "default": "String"
+                  "oneOf": [
+                    {
+                      "type": "string",
+                      "enum": [
+                        "String",
+                        "Number",
+                        "Boolean",
+                        "Object",
```

Comment:
```suggestion
```

I think Object should be removed.

Reply:
I'll be a breaking change for existing service, are you sure?